sdk/search/search-documents/test/public/node/searchClient.spec.ts

```diff
@@ -14,26 +13,55 @@ import {
   KnownQueryLanguage,
   KnownSpeller,
   SearchClient,
+  SearchIndex,
   SearchIndexClient,
   SelectFields,
 } from "../../../src";
 import { SearchFieldArray, SelectArray } from "../../../src/indexModels";
-import { delay, serviceVersions } from "../../../src/serviceUtils";
+import { defaultServiceVersion, delay } from "../../../src/serviceUtils";
 import { COMPRESSION_DISABLED } from "../../compressionDisabled";
 import { Hotel } from "../utils/interfaces";
 import { createClients } from "../utils/recordedClient";
 import { createIndex, createRandomIndexName, populateIndex, WAIT_TIME } from "../utils/setup";
 
-versionsToTest(serviceVersions, {}, (serviceVersion, onVersions) => {
-  onVersions({ minVer: "2023-11-01" }).describe("SearchClient tests", function (this: Suite) {
+describe("SearchClient", function (this: Suite) {
+  this.timeout(99999);
+
+  describe("constructor", function () {
+    const credential = new AzureKeyCredential("key");
+
+    describe("Passing serviceVersion", () => {
+      it("supports passing serviceVersion", () => {
+        const client = new SearchClient<Hotel>("", "", credential, {
+          serviceVersion: defaultServiceVersion,
+        });
+        assert.equal(defaultServiceVersion, client.serviceVersion);
+        assert.equal(defaultServiceVersion, client.apiVersion);
+      });
+
+      it("supports passing the deprecated apiVersion", () => {
```

Comment:
is this test correct? it looks like we're passing the defaultServiceVersion here as well - should we pass a deprecated one?